### ISSUE TITLE
Can't submit entries when not logged in.

### ISSUE THREAD

**Issue by jaydensmith** [barrelstrength/craft-sprout-forms#305]
### Steps to reproduce

1. Setup stock Craft 3.2.0-RC2 instance
2. Install Sprout Forms
3. Create form and add to frontend
4. Log out of CMS
5. Submit form

### Additional info

- Craft version: 3.2.0-RC2
- PHP version: 7.1.26
- Database driver & version:  MySQL 5.7.25
- Plugins & versions: Sprout Forms 3.2.1

```
craft\web\ServiceUnavailableHttpException in /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/craftcms/cms/src/web/Controller.php:162
Stack trace:
#0 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/yiisoft/yii2/base/Controller.php(155): craft\web\Controller->beforeAction(Object(yii\base\InlineAction))
#1 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/craftcms/cms/src/web/Controller.php(186): yii\base\Controller->runAction('save-entry', Array)
#2 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('save-entry', Array)
#3 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/craftcms/cms/src/web/Application.php(298): yii\base\Module->runAction('sprout-forms/en...', Array)
#4 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/craftcms/cms/src/web/Application.php(565): craft\web\Application->runAction('sprout-forms/en...', Array)
#5 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/craftcms/cms/src/web/Application.php(277): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#6 /Users/jaydensmith/public_html/discoveryelc-com-au/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#7 /Users/jaydensmith/public_html/discoveryelc-com-au/public_html/index.php(34): yii\base\Application->run()
#8 {main}
```

**Comment by jaydensmith** [comment]
Fix seems to be to a change to `EntriesController.php`:

```php
protected $allowAnonymous = [
    'save-entry'
];
```

becomes

```php
protected $allowAnonymous = [
    'save-entry' => true
];
```

**Comment by BenParizek** [comment]
Thanks @jaydensmith 

We'll look into this and get this resolved this coming week.

**Comment by BenParizek** [comment]
@jaydensmith This is an issue introduced by a change in 3.2. We'll have a fix out with 3.2 next week.

**Comment by jaydensmith** [comment]
@BenParizek Craft 3.2 is out of beta, do you have an ETA on this issue being resolved?

**Comment by BenParizek** [comment]
Hey @jaydensmith 

Upgrade to the latest and it should be resolved. We pushed a fix (v3.3.1) the same day as 3.2 was released and I just overlooked updating this ticket. Thanks for reporting.
